scale/job/test/test_models.py

```diff
@@ -18,11 +18,11 @@
 from job.configuration.data.job_data import JobData
 from job.configuration.interface.error_interface import ErrorInterface
 from job.configuration.interface.job_interface import JobInterface
-from job.execution.container import get_job_exe_input_vol_name, get_job_exe_output_vol_name, SCALE_JOB_EXE_INPUT_PATH, \
-    SCALE_JOB_EXE_OUTPUT_PATH
+from job.execution.container import get_job_exe_input_vol_name, get_job_exe_output_vol_name, \
+    get_workspace_volume_name, SCALE_JOB_EXE_INPUT_PATH, SCALE_JOB_EXE_OUTPUT_PATH
```

Comment:
Instead of importing all these individual functions, I think it's clearer to import the container namespace. You can still do a separate from import for the constants if you like. Also would prefer () line continuation instead of \.